rand_jitter/src/error.rs

```diff
@@ -10,24 +10,28 @@
 use rand_core::Error;
 use core::fmt;
 
+/// Base code for all `JitterRng` errors
+const ERROR_BASE: u32 = 0xAE53_0400;
```

Comment:
Maybe write it like `const ERROR_BASE: u32 = Error::INTERNAL_START + <some const>`?

Reply:
Is this actually clearer? To me it's not.